Source/Scene/Cesium3DTileset.js

```diff
@@ -1858,12 +1914,44 @@ define([
                 normal = Cartesian3.multiplyByScalar(normal, 0.75 * radius, scratchCartesian2);
                 position = Cartesian3.add(normal, boundingVolume.center, scratchCartesian2);
             }
-            tileset._geometricErrorLabels.add({
-                text: tile.geometricError.toString(),
-                position: position
+
+            var labelString = "";
+            var attribCounter = 0;
+
+            if (tileset.debugShowGeometricError) {
+                labelString += "\nGeometric error: " + tile.geometricError.toString();
+                attribCounter++;
+            }
+            if (tileset.debugShowNumberOfCommands) {
+                labelString += "\nCommands: " + tile.content.commandsLength.toString();
+                attribCounter++;
+            }
+            if (tileset.debugShowNumberOfPoints) {
+                labelString += "\nPoints: " + tile.content.pointsLength.toString();
+                attribCounter++;
+            }
+            if (tileset.debugShowNumberOfTriangles) {
+                labelString += "\nTriangles: " + tile.content.trianglesLength.toString();
+                attribCounter++;
+            }
+            if (tileset.debugShowTextureMemoryUsage) {
+                labelString += "\nTexture Memory: " + tile.content.textureMemorySizeInBytes.toString();
+                attribCounter++;
+            }
+            if (tileset.debugShowVertexMemoryUsage) {
+                labelString += "\nVertex Memory: " + tile.content.vertexMemorySizeInBytes.toString();
+                attribCounter++;
+            }
+
+            tileset._tileInfoLabels.add({
+                text: labelString.substring(1),
```

Comment:
Be consistent with the :'s. Usually the convention is space on both sides (`text : labelString.substring(1),`)